src/app/component/MissionGroup/MissionGroup.jsx

```diff
@@ -55,6 +55,14 @@ const MissionGroup = ({ callToAction, group, groupCallToAction, showViewRoute })
   };
   const onClickViewRouteButton = () => {
     console.log("View Routes");
+    missions.forEach((mission) => {
+      console.log(mission);
+      let pickUpLocation = mission.pickUpLocation.address;
+      let deliveryLocation = mission.deliveryLocation.address;
```

Comment:
We probably shouldn't use `let` for declarations if we're not reassigning these variables in this block scope.

Reply:
also no longer there